pkg/serveswagger/swagger_test.go

```diff
@@ -0,0 +1,44 @@
+package serveswagger
+
+import (
+	"os/exec"
+	"regexp"
+	"strings"
+	"testing"
+)
+
+func TestCheckTimestamps(t *testing.T) {
+	cmdGenerate := exec.Command("go", "generate", "./cmd/go-swagger/restapi")
```

Comment:
This is going to run in the directory this file is in, so it won't work here with a relative path. The fact that it's passing anyway tells me we want to assert that this actually does something and make the test fail if it doesn't. A simple check would be to capture stderr and make sure it's not empty:

```
var stderr bytes.Buffer
cmdGenerate.Stderr = &stderr
if err := cmdGenerate.Run(); err != nil{
	t.Fatal(err.Error())
}
if stderr.Len == 0 {
    t.Fatal(...)
}
```